scripts/pre_commit_linter.py

```diff
@@ -611,6 +626,8 @@
 _STDOUT_LIST = multiprocessing.Manager().list()
 _FILES = multiprocessing.Manager().dict()
 
+MAX_CONCURRENT_RUNS = 6
```

Comment:
This is more of a global number, I'd recommend putting this in the `semaphore_method` file. The max is whatever is in the original backend test file.

Reply:
Done.